src/acceptance/administrator/components/com_users/UserListCest.php

```diff
@@ -19,10 +19,10 @@ class UserListCest
 
 	public function __construct()
 	{
-		$this->username = "testUser";
-		$this->password = "test";
-		$this->name = "Test Bot";
-		$this->email = "Testbot@example.com";
+		$this->username = "testUser2";
```

Comment:
how does this help?, the problem is with Deletion of Data in the previous test I think